packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -15,6 +15,7 @@ import { buildAadManifest } from "./utility/buildAadManifest";
 import { UpdateAadAppOutput } from "./interface/updateAadAppOutput";
 import { InvalidActionInputError, UnhandledError, UnhandledUserError } from "../../../error/common";
 import { updateProgress } from "../middleware/updateProgress";
+import { ViewAadAppHelpLink } from "../../constants";
```

Comment:
The link is for V4, should we create a new link for V5?

Reply:
add a new link for v5